package.json

```diff
@@ -8,6 +8,7 @@
     "i18n_extract": "BABEL_ENV=i18n $(npm bin)/babel src --quiet > /dev/null",
     "i18n_cleanJson": "node scripts/cleanI18nJson.js",
     "lint": "eslint --ext .js --ext .jsx .",
+    "lint_fix": "eslint --fix --ext .js --ext .jsx .",
```

Comment:
I also added this command and ran it, that's why a few of these seemingly unrelated files are in the diff.